minio/minio
# Populate end_session_endpoint

## Description
Populates end_session_endpoint using configuration response from IDP provider

## Motivation and Context
https://github.com/minio/console/issues/2415

## How to test this PR?
* Setup keycloak provider: https://www.keycloak.org/getting-started/getting-started-docker
* Set and use IDP provider in minio: https://min.io/docs/minio/linux/operations/external-iam/configure-openid-external-identity-management.html#
* Login in Console
* Logout (It should end the session in keycloak
* Login again (New login should ask for credentials again)

## Types of changes

## Checklist:
